frontend/src/app/components/filters/filter-toggled-multiselect-value/filter-toggled-multiselect-value.component.html

```diff
@@ -1,47 +1,17 @@
-<div class="inline-label"
+<div class="inline-label ng-select-container"
      id="div-values-{{filter.id}}">
 
-  <select filter-value-select
-          *ngIf="!isMultiselect"
-          [(ngModel)]="value"
-          [compareWith]="compareByHrefOrString"
-          id="values-{{filter.id}}"
-          name="v[{{filter.id}}][]"
-          class="advanced-filters--select">
-    <option [textContent]="text.placeholder"
-            *ngIf="hasNoValue"
-            [ngValue]="null"
-            disabled></option>
-    <option *ngFor="let value of availableOptions"
-            [textContent]="value.name"
-            [ngValue]="value"></option>
-  </select>
-
-  <select multiple
-          filter-value-select
-          *ngIf="isMultiselect"
-          [(ngModel)]="value"
-          [compareWith]="compareByHrefOrString"
-          id="values-{{filter.id}}"
-          name="v[{{filter.id}}][]"
-          class="advanced-filters--select">
-    <option [ngValue]="null"
-            [textContent]="text.placeholder"
-            disabled
-            selected
-            *ngIf="hasNoValue"></option>
-    <option *ngFor="let value of availableOptions"
-            [textContent]="value.name"
-            [ngValue]="value"></option>
-  </select>
-
-  <a class="form-label filter-toggled-multiselect--toggler no-decoration-on-hover -transparent"
-     (click)="toggleMultiselect()">
-    <op-icon *ngIf="isMultiselect"
-             icon-classes="icon4 icon-minus2"
-             [icon-title]="text.disableMulti"></op-icon>
-    <op-icon *ngIf="!isMultiselect"
-             icon-classes="icon4 icon-add"
-             [icon-title]="text.enableMulti"></op-icon>
-  </a>
+  <ng-select [(ngModel)]="value"
```

Comment:
Please add the option ` [clearSearchOnAdd]="true"` to remove the search text after the user selected an entry.

Reply:
Done!